x/examples/website-wrapper-app/wrapper_app_project/.scripts/build.mjs

```diff
@@ -48,16 +39,29 @@ export default async function main(
   {
     platform,
     sdkVersion = "x/v0.0.1",
-    entryDomain = "www.example.com",
+    additionalDomains = [],
+    appId,
+    appName,
+    entryUrl = "https://www.example.com",
     navigationUrl,
+    output = OUTPUT_DIR,
+    platform,
     smartDialerConfig = DEFAULT_SMART_DIALER_CONFIG,
-    additionalDomains = [],
   },
 ) {
+  const WRAPPER_APP_OUTPUT_DIR = path.resolve(output, "wrapper_app_project");
+  const WRAPPER_APP_OUTPUT_ZIP = path.resolve(output, "wrapper_app_project.zip");
+  
+  const SDK_MOBILEPROXY_OUTPUT_DIR = path.resolve(output, "mobileproxy");
+  const WRAPPER_APP_OUTPUT_SDK_MOBILEPROXY_DIR = path.resolve(
+    WRAPPER_APP_OUTPUT_DIR,
+    "mobileproxy",
+  );
+
   if (!fs.existsSync(SDK_MOBILEPROXY_OUTPUT_DIR)) {
     console.log(`Building the Outline SDK mobileproxy library for ${platform}...`);
 
-    await promisify(exec)(`npm run build:sdk_mobileproxy ${platform} ${sdkVersion}`);
+    await promisify(exec)(`npm run build:mobileproxy ${platform} ${sdkVersion} ${output}`);
```

Comment:
This is not quite safe. Can you use the api that takes the arguments separately instead of a full command line as a string?